### ISSUE TITLE
Time until the restriction is lifted

### ISSUE THREAD

**Issue by gurland** [telegramdesktop/tdesktop#4650]
### Steps to reproduce
1. Get Read-Only restriction
2. Look at the bottom of main window

### Actual behaviour
I am seing that message: "The admins of this group restricted you from writing here."

### Expected behaviour
But I'm not permanently restricted so I'm expecting to see message which contains information about time of restriction removal (that's implemented in mobile clients)

### Configuration
**Operating system:**
Windows 10

**Version of Telegram Desktop:**
1.2.17

**Used theme**:
Default (night mode)

**Comment by NTMan** [comment]
Yes, please make it as it already implemented in android client.

![Screenshot_20190614_101104_org telegram messenger](https://user-images.githubusercontent.com/200750/59487191-4e832680-8e95-11e9-9ea4-6344153990f6.jpg)

**Comment by NTMan** [comment]
issue is actual

**Comment by NTMan** [comment]
the feature request still actual
